src/openrct2/ride/RideData.h

```diff
@@ -328,114 +328,126 @@ struct TrackDrawerDescriptor
 
 enum class RtdFlag : uint8_t
 {
-    hasTrackColourMain = 0,
```

Comment:
Any particular reason for removing the explicit numbering?

Reply:
It makes adjusting the flags cumbersome, and it didn’t serve a purpose. (It used to, when the flags had built-in left shifts, I just forgot to remove it when I changed them earlier this year.) It’s also never serialised.

Reply:
Hmm. I suppose they'll only be serialised with a string intermediary in the objects, eventually, right? I'm still partial to numbered enums for debugging purposes, tbh. I'm not going to block it, though.